docs/artifactory/pipdepstree/help.go

```diff
@@ -0,0 +1,8 @@
+package pipdepstree
+
+const Description = "Show pip dependency tree."
+
+var Usage = []string{`jfrog rt pdt <pip command>`}
+
+const Arguments string = `	pip command
+		Arguments and options for the pip-install command.`
```

Comment:
...for the pip-install command. --> ...for the pip command.
(should be also fixed in *pipinstall/help.go*).